docs/source/options/general_encryption_options.rst

```diff
@@ -13,6 +13,9 @@ General encryption options
 --clear_lead <seconds>
 
     Clear lead in seconds if encryption is enabled.
+    Shaka Packager does not support partial encrypted segments, so if segment_duration
+    is 6 seconds, then only the first segment is in clear (without encryption), with all the
```

Comment:
This is only true if the value of clear_lead is smaller than segment_duration. Can you change it to something like:

so if segment_duration is greater than clear_lead, then only the first segment is in clear (without encryption) with all the following segments encrypted.